onnxruntime/core/framework/tuning_context.h

```diff
@@ -28,6 +28,10 @@ class ITuningContext {
   virtual void DisableTuning() = 0;
   virtual bool IsTuningEnabled() const = 0;
 
+  virtual void SetMaxTuningDurationMs(int max_duration_ms) = 0;
+  virtual int GetMaxTuningDurationMs() const = 0;
+  virtual bool IsMaxTuningDurationMsValid() const = 0;
```

Comment:
`IsMaxTuningDurationMsValid` is not proper, you should handle the invalid value on set, say, if `max_duration_ms <=0`, then convert it to `INT_MAX`.

Reply:
Or, on get